ejb/WFLY-13828-remote+tls-is-not-supported-by-EJBClient-and-remote-outbound-connection.adoc

```diff
@@ -0,0 +1,49 @@
+= remote+tls is not supported by EJBClient and remote-outbound-connection
+:email:             tomasz.adamski@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+=== Issue
+* https://issues.redhat.com/browse/WFLY-13828
```

Comment:
At the moment the analysis is only mentioning changes which would need to be applied to the Remoting subystem which lived in WildFly Core so we probably need a WFCORE as well.

Reply:
@darranl WFCORE jira is mentioned in next line: https://issues.redhat.com/browse/WFCORE-5718

Can we close this thread?